static/app/views/performance/browser/resources/utils/useResourceDomansQuery.ts

```diff
@@ -0,0 +1,48 @@
+import {useDiscoverQuery} from 'sentry/utils/discover/discoverQuery';
+import EventView from 'sentry/utils/discover/eventView';
+import {DiscoverDatasets} from 'sentry/utils/discover/types';
+import {useLocation} from 'sentry/utils/useLocation';
+import useOrganization from 'sentry/utils/useOrganization';
+import usePageFilters from 'sentry/utils/usePageFilters';
+import {useResourceModuleFilters} from 'sentry/views/performance/browser/resources/utils/useResourceFilters';
+
+/**
+ * Gets a list of pages that have a resource.
+ */
+export const useResourceDomainsQuery = () => {
+  const location = useLocation();
+  const pageFilters = usePageFilters();
+  const {slug: orgSlug} = useOrganization();
+  const {transaction} = useResourceModuleFilters();
+
+  const fields = ['avg(span.self_time)', 'span.domain']; // TODO: this query fails without avg(span.self_time)
```

Comment:
m: can we use `SpanMetricsField.SPAN_DOMAIN`? Likewise with all other cases.

I think we need an aggregation like `avg`. We used `count()` for the domain selector on the database page. Not sure what is faster though 🤔.

Reply:
Intuitively count sounds like it would be faster, because the simplest calculation for average includes count() avg = sum(duration)/count(). But maybe clickhouse does something under the hood 🤷 
I'll switch this to count just to be consistent